packages/kit/src/cli.js

```diff
@@ -81,6 +82,7 @@ prog
 
 		process.env.NODE_ENV = 'development';
 		const config = await get_config();
+		const basePath = config.kit.paths.base;
```

Comment:
mind replacing all the `basePath` occurrences? we don't use camelCase in this codebase

```suggestion
		const { base } = config.kit.paths;
```

Reply:
Force of habit 😅. Done.